apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx

```diff
@@ -46,37 +47,60 @@ export function PatientOverview({
     return <IDSAlert>{sjfMetaData.haveSekretess ? PROTECTED_PERSON_ALERT : PERSON_RESPONSE_MISSING_ALERT}</IDSAlert>
   }
 
-  return (
-    <div className="grid grid-cols-2 gap-4 py-5">
-      <OpenInformationCard
-        title="Ospärrad information inom egen vårdgivare"
-        subTitle="Vårdenhet med information"
-        description="Det finns ospärrad information hos en annan vårdenhet inom din vårdgivare. Du kan klicka nedan för att visa vilka vårdenheter som
-        har denna information och få möjlighet att inhämta den."
-        items={sjfMetaData.kraverInteSamtycke}
-        onGetInformation={handleGetCareUnitInformation}
-      />
-      <OpenInformationWithConsentCard
-        items={sjfMetaData.kraverSamtycke}
-        title="Ospärrad information hos annan vårdgivare"
-        subTitle="Vårdgivare med information"
-        description="Det finns ospärrad information hos en annan vårdgivare. Du kan klicka nedan för att visa vilka vårdgivare som har ospärrad information. Patientens samtycke krävs för att du ska kunna ta del av informationen."
-        hasGivenConsent={sjfMetaData.samtyckeFinns}
-        onGetInformation={handleGetCareGiverInformation}
-        onGiveConsent={handleGiveConsent}
-      />
-      <BlockedInformationCard
-        title="Spärrad information inom egen vårdgivare"
-        subTitle="Vårdenhet"
-        description="Det finns spärrad information hos en annan vårdenhet inom din vårdgivare. Endast patienten kan få spärren hävd genom att kontakta den enhet där spärren sattes. Du kan klicka nedan för att visa vilka vårdenheter som har spärrad information hos sig."
-        items={sjfMetaData.vardenheterInomVGMedSparr}
+  if (sjfMetaData.consentServiceError || sjfMetaData.blockingServiceError) {
+    return (
+      <ErrorModal
+        show={sjfMetaData.consentServiceError || sjfMetaData.blockingServiceError}
+        description={
+          sjfMetaData.consentServiceError
+            ? 'Tyvärr kan information från andra vårdgivare inte hämtas på grund av ett tekniskt fel. Försök igen om en stund.'
```

Comment:
Is this text correct? According to the "krav" it should be

```Information från andra vårdgivare kan inte hämtas på grund av ett tekniskt fel. Försök igen om en stund. Om problemet kvarstår, kontakta i första hand din lokala IT-support och i andra hand <Inera Support>.```

Reply:
Looks like it has updated while the PR has been open, I'll fix the texts

Reply:
"Något har blivit fel med anrop till Spärrtjänsten. Felet ligger hos dem, inte oss." is when blockingServiceError is true